feature-detects/css/pointerevents.js

```diff
@@ -21,10 +21,10 @@
   }]
 }
 !*/
-define(['Modernizr', 'createElement'], function(Modernizr, createElement) {
+define(['Modernizr', 'setCss', 'createElement'], function(Modernizr, setCss, createElement) {
   Modernizr.addTest('csspointerevents', function() {
-    var style = createElement('a').style;
-    style.cssText = 'pointer-events:auto';
-    return style.pointerEvents === 'auto';
+    var elem = createElement('a');
```

Comment:
```suggestion
    var elem = createElement('x');
```
The other pr is creating an `<x>` element, again, not their change.  What even is an `<x> `element?